src/snapred/ui/workflow/NormalizationCalibrationWorkflow.py

```diff
@@ -190,31 +196,61 @@ def callNormalizationCalibration(self, groupingFile, smoothingParameter):
 
         self._specifyNormalizationView.updateWorkspaces(focusWorkspace, smoothWorkspace)
 
-    def onNormalizationValueChange(self, index, smoothingValue):  # noqa: ARG002
+    def applySmoothingUpdate(self, index, smoothingValue, dMin):
+        from mantid.simpleapi import DeleteWorkspace
+
+        workspaces = self.responses[-1].data
+        DeleteWorkspace(Workspace=workspaces["smoothedOutput"])
+        payload = SmoothDataExcludingPeaksRequest(
+            inputWorkspace=workspaces["outputWorkspace"],
+            outputWorkspace=workspaces["smoothedOutput"],
+            samplePath=self.samplePaths[self.sampleIndex],
+            groupingPath=self.groupingFiles[index],
+            runNumber=self.runNumber,
+            smoothingParameter=smoothingValue,
+            dMin=dMin,
+        )
+        request = SNAPRequest(path="normalization/smooth", payload=payload.json())
+        response = self.interfaceController.executeRequest(request)
+        focusWorkspace = workspaces["outputWorkspace"]
+        smoothWorkspace = response.data
+        self.responses[-1].data["smoothedOutput"] = response.data
+        self._specifyNormalizationView.updateWorkspaces(focusWorkspace, smoothWorkspace)
+
+    def onNormalizationValueChange(self, index, smoothingValue, dMin):  # noqa: ARG002
         if not self.initializationComplete:
             return
+        # disable recalculate button
+        self._specifyNormalizationView.disableRecalculateButton()
 
         self.lastGroupingFile = self.groupingFiles[index]
         self.lastSmoothingParameter = smoothingValue
+        self.lastDMin = dMin
 
         groupingFileChanged = self.groupingFiles[self.initGroupingIndex] != self.lastGroupingFile
         smoothingValueChanged = self.initSmoothingParameter != self.lastSmoothingParameter
-
-        if groupingFileChanged or smoothingValueChanged:
+        dMinValueChanged = self.initDMin != self.lastDMin
+        if groupingFileChanged:
             from mantid.simpleapi import DeleteWorkspace
 
-            DeleteWorkspace(Workspace="focussedRawVanadium")
-            DeleteWorkspace(Workspace="smoothedOutput")
+            workspaces = self.responses[-1].data
+
+            # BAD! >:C This shouldnt be here. This should be in the backend
+            DeleteWorkspace(Workspace=workspaces["outputWorkspace"])
+            DeleteWorkspace(Workspace=workspaces["smoothedOutput"])
```

Comment:
use grocery service's `deleteWorkspaceUnconditional`

Reply:
UI files really shouldnt be interacting with mantid directly at all, I just wanted to limit the scope of my refactor and use what was already implemented.  I think the right refactor would be at and another backend point for deleting workspaces...

btw does groceryService account for users deleting workspaces through the ui?

Reply:
Actually I can just delete these lines, because if the backend is setup correctly they workspaces will just be overwritten anway

Reply:
I remember that these lines have to be here or else something will crash.
GroceryService does check against the mantid ADS to make sure its insync, but admittedly it does so imperfectly